sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsAutoConfiguration.java

```diff
@@ -56,10 +57,17 @@ ServiceBusJmsConnectionFactoryCustomizer amqpOpenPropertiesCustomizer() {
         };
     }
 
+    /**
+     * The BeanPostProcessor to instrument the {@link AzureServiceBusJmsProperties} bean with provided connection string
+     * providers.
+     * @param connectionStringProviders the connection string providers to provide the Service Bus connection string.
+     * @return the bean post processor.
+     */
     @Bean
     @ConditionalOnMissingBean
     @ConditionalOnMissingProperty(prefix = "spring.jms.servicebus", name = "connection-string")
-    public static AzureServiceBusJmsPropertiesBeanPostProcessor azureServiceBusJmsPropertiesBeanPostProcessor(ObjectProvider<ConnectionStringProvider<AzureServiceType.ServiceBus>> connectionStringProviders) {
+    public static AzureServiceBusJmsPropertiesBeanPostProcessor azureServiceBusJmsPropertiesBeanPostProcessor(
+        ObjectProvider<ConnectionStringProvider<AzureServiceType.ServiceBus>> connectionStringProviders) {
```

Comment:
why arg name use plural style?

Reply:
this is not the change of this pr, but when you use ObjectProvider, it means there could be multiple such objects.